lib/jdomodels/src/test/java/org/sagebionetworks/repo/model/dbo/dao/migration/MigratableTableDAOImplAutowireTest.java

```diff
@@ -121,6 +126,27 @@ public void testMigrationRoundTrip() throws Exception {
 		assertEquals(preview2.getEtag(), row.getEtag());
 		assertEquals(null, row.getParentId());
 		
+		// Get migration type count

```

Comment:
Move to own test and don't depend on other methods.
Start with empty table and check as rows are added.